internal/config/tracing.go

```diff
@@ -22,11 +22,10 @@ type TracingConfig struct {
 func (c *TracingConfig) setDefaults(v *viper.Viper) error {
 	v.SetDefault("tracing", map[string]any{
 		"enabled":  false,
-		"exporter": TracingJaeger,
+		"exporter": TracingOTLP,
```

Comment:
Questioning this incase it wasn't considered:
Should we be simultaneously deprecating and changing the default behaviour like this?

Just wondering if there might be folks out there with solely:
```yaml
tracing:
  enabled: true
```
That might just be relying on it talking to jaeger.

Reply:
good call. I also added a `tracing` section to telemetry so we can hopefully get more data about when its safe to fully remove